### PR TITLE
[New Package] pyconfigure

### PR DESCRIPTION
**Pull request by saltedcoffii**

### CODE REVIEW

File: packages/pyconfigure.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require 'package'
+
+class Pyconfigure < Package
+  description 'GNU pyconfigure provides developers with file templates for implementing standard configure scripts and Makefile recipes for their Python packages.'
+  homepage 'https://www.gnu.org/software/pyconfigure/'
+  version '3.1.1'
```
Comment (uberhacker) [inline packages/pyconfigure.rb:6]: Doesn't match source url version.

File: packages/pyconfigure.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  compatibility 'all'
+  source_url 'https://ftpmirror.gnu.org/pyconfigure/pyconfigure-0.2.3.tar.gz'
+  source_sha256 '37438ee1ae0aad7692188c7d01218cb407a1c89f3efdb5d360311b7ade4ab14f'
+  
+  depends_on 'python2_7'
```
Comment (uberhacker) [inline packages/pyconfigure.rb:11]: `depends_on 'python27'`?

Comment (uberhacker) [reply packages/pyconfigure.rb:11]: Actually, you can just exclude since it's a core package.

File: packages/pyconfigure.rb (hunk 1 above)
Comment (saltedcoffii) [reply packages/pyconfigure.rb]: done

File: packages/pyconfigure.rb (hunk 2 above)
Comment (saltedcoffii) [reply packages/pyconfigure.rb]: done

**Pull request merged**